components/FilterByActivity/FilterByActivity.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text, Button, TouchableOpacity} from 'react-native';
+import ModalDropdown from 'react-native-modal-dropdown';
+
+export default function FilterByActivity(props) {
+
+  const [ activityFilterValue, setActivityFilterValue ] = useState('')
+
+  const setFilterValue = (value) => {
+    //  setActivityFilterValue(value)
+    props.setFilter(value)
+  }
+  return (
+    <View style={styles.dropdownContainer}>
+    <ModalDropdown
+      renderButtonText={activityFilterValue => setFilterValue(activityFilterValue)}
+      textStyle={styles.dropdownText}
+      dropdownTextStyle={styles.dropdownTextStyle}
+      defaultValue='By activity -'
+      options={[ 'Show all events', 'Baseball', 'Basketball', 'Biking', 'Bocce ball', 'Climbing', 'Croquet', 'Disc golf', 'Fishing', 'Football', 'Hand ball', 'Kickball', 'Outdoor exercising', 'Slacklining', 'Soccer', 'Spike ball', 'Tennis', 'Trail hiking/walking', 'Ultimate frisbee', 'Volleyball', 'Miscellaneous']}
```

Comment:
I think this is a great start for our activity list. Bonus points for putting it in alphabetical order!